plugins/npm/npm.go

```diff
@@ -0,0 +1,29 @@
+package npm
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func NPMCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "NPM install/publish",
+		Runs:      []string{"npm"},
+		DocsURL:   sdk.URL("https://docs.npmjs.com/cli"),
+		NeedsAuth: needsauth.IfAny(
+			needsauth.WhenContainsArgs("i"),
+			needsauth.WhenContainsArgs("install"),
```

Comment:
I don't know if it should be supported, but npm install has a lot of different alias.

> npm install [<package-spec> ...]
> aliases: add, i, in, ins, inst, insta, instal, isnt, isnta, isntal, isntall
> https://docs.npmjs.com/cli/v9/commands/npm-install

Reply:
Yes, I would say all aliases should be included here, otherwise the plugin's security guarantees could simply be bypassed by using an alias.

Reply:
>  the plugin's security guarantees could simply be bypassed by using an alias

Could you expand on that? I'm not sure I understand why that would be the case.